misc/extension_api_validation/4.2-stable.expected

```diff
@@ -147,3 +147,15 @@ GH-84792
 Validate extension JSON: Error: Field 'classes/RenderingServer/methods/environment_set_fog/arguments': size changed value in new API, from 10 to 11.
 
 Added fog mode argument. Compatibility method registered.
+
+
+GH-81746
+--------
+Validate extension JSON: API was removed: classes/GLTFDocument/methods/append_from_buffer
+Validate extension JSON: API was removed: classes/GLTFDocument/methods/append_from_file
+Validate extension JSON: API was removed: classes/GLTFDocument/methods/append_from_scene
+Validate extension JSON: API was removed: classes/GLTFDocument/methods/generate_buffer
+Validate extension JSON: API was removed: classes/GLTFDocument/methods/generate_scene
+Validate extension JSON: API was removed: classes/GLTFDocument/methods/write_to_filesystem
```

Comment:
These are breaking changes in GDExtension, and afaik `GLTFDocument` is a stable (not experimental) API.

Reply:
@lyuma I will revert to the changes earlier then unless you can submit a pr to change it.